doc/content/getting-started/aws/ecs/troubleshooting/_index.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Troubleshooting AWS ECS Deployment"
+description: ""
+weight: 7
+---
+
+<!--
+TODO: https://github.com/TheThingsNetwork/lorawan-stack/issues/2714
+Move to generic getting started guide once ready.
+-->
+
+This section contains information to troubleshoot {{% tts %}} ECS deployment.
+
+<!--more-->
+
+### TTS no longer accepting connections from UDP gateways after upgrading to latest version
```

Comment:
This is a bit vague and makes it seems like new TTS versions are buggy. I'd rephrase this as
```
UDP gateways not connected after an update.
```